src/bit_string.rs

```diff
@@ -40,18 +41,18 @@ impl<T> SegmentOptionCategories<'_, T> {
 
     fn segment_type(&self) -> Arc<Type> {
         match self.typ {
-            Some(BitStringSegmentOption::Int { .. }) => crate::type_::int(),
-            Some(BitStringSegmentOption::Float { .. }) => crate::type_::float(),
-            Some(BitStringSegmentOption::Binary { .. }) => crate::type_::bit_string(),
-            Some(BitStringSegmentOption::BitString { .. }) => crate::type_::bit_string(),
-            Some(BitStringSegmentOption::Utf8 { .. }) => crate::type_::string(),
-            Some(BitStringSegmentOption::Utf16 { .. }) => crate::type_::string(),
-            Some(BitStringSegmentOption::Utf32 { .. }) => crate::type_::string(),
-            Some(BitStringSegmentOption::Utf8Codepoint { .. }) => crate::type_::utf_codepoint(),
-            Some(BitStringSegmentOption::Utf16Codepoint { .. }) => crate::type_::utf_codepoint(),
-            Some(BitStringSegmentOption::Utf32Codepoint { .. }) => crate::type_::utf_codepoint(),
+            Some(Int { .. }) => crate::type_::int(),
```

Comment:
Could we move the `use` into this function? I would prefer to have unqualified imports be as restricted as possible to make it clear where things are defined.

Reply:
My aim was mostly to shorten the matches further below that use or-patterns, because otherwise rustfmt does weird things like
```rust
Struct(
     field: Some(
         op
         @
         LongEnum::VariantA { .. }
         | LongEnum::VariantB { .. }
))
```
so maybe I could move the use only there?

Reply:
I'm happy with it being unqualified, just would like the `use Enum::*` to be inside the module rather than at the top level
https://github.com/gleam-lang/gleam/pull/1159/files#diff-259329068baeb3eee63257f180ea5aff88a8ed160bb3ed2d14dc10fae02d7b1bR1